pkg/sdkserver/sdkserver_test.go

```diff
@@ -901,6 +903,83 @@ func TestSDKServerReserveTimeout(t *testing.T) {
 	wg.Wait()
 }
 
+/* Alpha Functionality */
+
+func TestSDKServerPlayerCapacity(t *testing.T) {
```

Comment:
There are a couple of comments that "if the player capacity is set from outside the SDK, use SDK.GameServer() instead." It would be nice to have a test that proves the comment true.

Reply:
Thanks for bringing this up - I wasn't sure the best approach here.

So I deliberately haven't implemented the Alpha properties on SDK.GameServer() because I'm not sure what the right approach here is (do we have a separate SDK.Alpha().GameServer()? can we integrate it into the stable version? Each has trade offs), so I wrote a comment of what I figured might make sense, with intent to change it later on once we hashed out the details in a separate PR/discussion.

I'm wondering if there should be a `TODO:` of some kind here for this PR, so it's clear this isn't implemented functionality that needs to be updated later on? Would that work?

Reply:
A TODO would be fine since this is alpha and there are some unanswered questions.